framework/src/Volo.Abp.Auditing/Volo/Abp/Auditing/JsonNetAuditSerializer.cs

```diff
@@ -18,10 +20,29 @@ public string Serialize(object obj)
         {
             var options = new JsonSerializerSettings
             {
-                ContractResolver = new AuditingContractResolver(Options.IgnoredTypes)
+                ContractResolver = GetSharedAuditingContractResolver(Options.IgnoredTypes)
             };
 
             return JsonConvert.SerializeObject(obj, options);
         }
+
+        private static readonly object SyncObj = new object();
+        private static AuditingContractResolver _sharedAuditingContractResolver;
+
+        public static AuditingContractResolver GetSharedAuditingContractResolver(List<Type> ignoredTypes)
```

Comment:
Make this non static and directly use Options.IgnoredTypes instead of getting it as argument. This would be a better code design.